operator/v1/types_console.go

```diff
@@ -143,8 +143,42 @@ type Capability struct {
 	Visibility CapabilityVisibility `json:"visibility"`
 }
 
+// ThemeType defines the type of the theme for the console UI.
+type ThemeType string
+
+// ThemeType values
+const (
+	// DarkTheme represents the dark theme for the console UI.
+	DarkTheme ThemeType = "Dark"
+	// LightTheme represents the light theme for the console UI.
+	LightTheme ThemeType = "Light"
+)
+
+// CustomFiles defines a configuration based on theme types for the console UI logo.
+type CustomFiles struct {
+	// type of the theme for the console UI.
```

Comment:
The documentation on this field is pretty sparse. I'd recommend taking a look at https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#write-user-readable-documentation-in-godoc and thinking about how you could improve the godoc here to answer some of the key questions in the doc I linked.